lib/Repository.js

```diff
@@ -491,7 +491,7 @@ class Repository extends Requestable {
     * @param {Requestable.callback} [cb] - will receive the pull request information
     * @return {Promise} - the promise for the http request
     */
-   updatePullRequst(number, options, cb) {
+   updatePullRequest(number, options, cb) {
```

Comment:
If you do an alias add a log that will be displayed regardless of the debug preferences that this is deprecated so you can remove it in subsequent major releases.